tests/lib/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -226,12 +226,12 @@ public static void MultidimensionalPrimitiveArray()
         }
 
         [Fact]
-        public static void ThrowsSystemObjectNotFoundException()
+        public void ThrowsSystemObjectNotFoundException()
         {
             var dependencyFilter = Substitute.For<IDependencyFilter>();
             dependencyFilter.IsFrameworkAssembly(Arg.Any<AssemblyReferenceInformation>()).Returns(false);
             var dependencyFinder = new ReflectionMetadataDependencyFinder(dependencyFilter, new SystemObjectFinder(dependencyFilter));
-            var assemblyToTest = TestAssembly.Create("FilterApis.cs", false, new[] { typeof(Image).GetTypeInfo().Assembly.Location });
```

Comment:
The additional reference isn't needed anymore?

Reply:
Nope. Not sure why it was there. We actually don't need it for anything.